packages/zoe/test/unitTests/zcf/test-zcf.js

```diff
@@ -1237,6 +1237,19 @@ test(`zcf.shutdownWithFailure - no further offers accepted`, async t => {
   t.truthy(vatAdminState.getExitWithFailure());
 });
 
+test(`zcf.assert - no further offers accepted`, async t => {
+  const { zoe, zcf, vatAdminState } = await setupZCFTest({});
```

Comment:
Do we need to mark this as Data?

```suggestion
  const { zoe, zcf, vatAdminState } = await setupZCFTest(Data({}));
```

Reply:
Looking at other uses of `setupZCFTest` it seems like an adequate answer is just to omit this argument and let it default. However, I fixed all that I found the same way. PTAL.